### PR TITLE
MT-347 : Update PhysicsEngine

### PR DESCRIPTION
**Pull request by RajashekarGoudpodeti**

### CODE REVIEW

**Comment by RajashekarGoudpodeti** [comment]
@nathangeorge1 can you approve and try now

File: src/simulation/src/simulation/PhysicsEngine.cpp [hunk 1]
```diff
@@ -124,19 +125,32 @@ Bird PhysicsEngine::updatePhysics(Bird bird, int curr_action, float time_interva
     // 2. find forces
     // gravity force
     float gravity_force = calcGravityForce(m);
-    F[2] += gravity_force; // gravity force
+    F[2] += gravity_force; 
+    
     ROS_DEBUG_STREAM("-----------gravity F: " << gravity_force);
 
     // calculating lift forces
```
Comment (void-robotics) [inline src/simulation/src/simulation/PhysicsEngine.cpp:130]: lines 132-156; spacing is off; see SP2

File: src/simulation/src/simulation/PhysicsEngine.cpp [hunk 2]
```diff
@@ -124,19 +125,32 @@ Bird PhysicsEngine::updatePhysics(Bird bird, int curr_action, float time_interva
...
-
-        F[0] += top_lift_force[0] + bottom_lift_force[0];
-        F[1] += top_lift_force[1] + bottom_lift_force[1];
-        F[2] += top_lift_force[2] + bottom_lift_force[2];
+        // calculating forces for upward,stable and downward movement
+        if(curr_action == 1) { 								// Upwards
+ 		F[0] += top_lift_force[0] + bottom_lift_force[0];
+		F[1] += top_lift_force[1] + bottom_lift_force[1];
+		F[2] += top_lift_force[2] + bottom_lift_force[2];
+	} else if (curr_action == -1) {							// stable
```
Comment (void-robotics) [inline src/simulation/src/simulation/PhysicsEngine.cpp]: see LO1

**Review by ghost — COMMENTED** [review]
I added inline comments; also remember to only use Jira for asking for PR merging; we do use the GitHub inline comments though; PR looks good; almost ready

**Review by ghost — APPROVED** [review]

**Pull request merged**